packages/checkbox/src/index.mts

```diff
@@ -96,9 +96,14 @@ export default createPrompt(
 
     const [active, setActive] = useState(bounds.first);
     const [showHelpTip, setShowHelpTip] = useState(true);
+    const [showValidationMessage, setShowValidationMessage] = useState(false);
 
     useKeypress((key) => {
       if (isEnterKey(key)) {
+        if (items.filter(isChecked).length === 0) {
+          setShowValidationMessage(true);
```

Comment:
Though this work for now, it'll not extend to new behaviour and custom validation logic.

I'd refer you to the [validation implementation inside the `input` prompt](https://github.com/SBoudrias/Inquirer.js/blob/master/packages/input/src/index.mts#L49). Using `setError(string)`, we'll have a more flexible validation feature.